src/components/Results/displayResults.js

```diff
@@ -9,27 +9,21 @@ function DisplayResults(props) {
   const analysedURL = resultsObj.url;
   console.log(resultsObj);
 
-  
-const article = resultsObj.article;
+  const article = resultsObj.article;
 
-
-  // temporary code to support credibility score while it's here
   const credibilityScore = resultsObj.results[0].outcome.score / 100;
   const source = resultsObj.results[0].outcome.source;
   const category = resultsObj.results[0].outcome.category;
 
   return (
     <div>
+      <ArticleSummary displayURL={analysedURL} displayArticle={article} />
```

Comment:
In relation to the note above, I wonder if passing the entire results object would be more useful in detecting an error:

`<Credibility outcome={resultsObj.results[0].outcome}>
`
I think this works fine in the meantime though.

Reply:
This is better! I'll go see what I can do.